.zuul.yaml

```diff
@@ -35,10 +35,46 @@
       - README*
       - .*/*.md
 
+- job:
+    name: functional-logging-tests-osp18
+    parent: telemetry-operator-multinode-autoscaling
```

Comment:
The parent job should be ``telemetry-operator-multinode-logging``, which is defined here: https://github.com/openstack-k8s-operators/telemetry-operator/blob/e586b1fdb90e05beb8c2a388d24184305cefbf8a/zuul.d/projects.yaml#L32

This job deploys the  logging features.

The cifmw_extras needs to be updated as well:
        - "@{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/ci-framework'].src_dir  }}/scenarios/centos-9/multinode-ci.yml"
        - "@{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/telemetry-operator'].src_dir }}/ci/vars-logging.yml"
        - "@{{ ansible_user_dir }}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}/ci/vars-logging-test.yml"

Reply:
Done